includes/class-wc-stripe-sepa-payment-token.php

```diff
@@ -105,4 +112,22 @@ public function get_last4( $context = 'view' ) {
 	public function set_last4( $last4 ) {
 		$this->set_prop( 'last4', $last4 );
 	}
+
+	/**
+	 * Set Stripe payment method type.
+	 *
+	 * @param string $type
+	 */
+	public function set_payment_method_type( $type ) {
+		$this->payment_method_type = $type;
+	}
```

Comment:
In case a SEPA token has been generated from a different payment method, we will be able to filter these SEPA tokens according to the corresponding payment methods that are enabled in the UPE.